pages/balance-budget.vue

```diff
@@ -364,4 +328,60 @@ export default Vue.extend({
   text-align: center;
   color: $dark-turquoise;
 }
+
+.Slider-Hint{
+    font-size: 18px;
+    font-weight: normal;
+    color: #ff5252;
+}
+
+.Health-Color{
```

Comment:
nit: seems like you could abstract most of this to a shared `color` class, e.g.
```
.color {
    height: 20px;
    width: 20px;
    border-radius: 50%;
}
```

then your colors would have `class="color Health-Color"` or something like that

Reply:
agreed + edited appropriately!